src/shared/utils.ts

```diff
@@ -166,22 +166,23 @@ export const languages = [
   { code: "lt" },
 ];
 
-export const themes = [
-  "litera",
-  "materia",
-  "minty",
-  "solar",
-  "united",
-  "cyborg",
-  "darkly",
-  "journal",
-  "sketchy",
-  "vaporwave",
-  "vaporwave-dark",
-  "i386",
-  "litely",
-  "nord",
-];
+export function fetchThemes(): Promise<string[]> {
```

Comment:
`export async function...`

and then when calling it do `await fetchThemes()...`